### DISCUSSION TITLE
Disable lualine for specific buffer type

### DISCUSSION THREAD

**Discussion by JPFrancoia** [nvim-lualine/lualine.nvim#1297] [Configuration]
Hi,

I'm migrating from airline to lualine, and there is just one tiny bit of config I can't get to work.

I have this in my vimrc:

```
"Needed to get vial-http to display information in status bar
"See: https://github.com/baverman/vial-http/issues/9
au WinEnter,BufWinEnter __vial_* let w:airline_disabled=1
```

Basically for some buffers, I want to disable airline/lualine. This is because *both* airline and lualine prevents another plugin (vial-http) to display some info in the status bar. I know it's a bit of a niche use case, but any help would be appreciated.

I tried playing with the the `disabled_filetypes`  and `disabled_buftypes`, but I couldn't really get what I wanted.

**Comment by iton0** [comment]
Check out this [section](https://github.com/nvim-lualine/lualine.nvim?tab=readme-ov-file#disabling-lualine) of the docs

**Reply by JPFrancoia** [reply]
I mentioned in my initial message that I have tried that. I can't disable by filetype, I would need to disable by "buffer type"

**Reply by iton0** [reply]
If you look at the whole section there is the `hide` method for disabling and reenabling lualine. You could implement that into your autocommand such that it will disable on buffer type.
